mesonbuild/scripts/coverage.py

```diff
@@ -68,6 +68,15 @@ def coverage(source_root, build_root, log_dir):
                                '--show-details',
                                '--branch-coverage',
                                covinfo])
+    if gcovr_exe:
+        print('')
+        print('XML coverage report can be found at',
+            pathlib.Path(log_dir, 'coverage.xml').as_uri())
+        print('Text coverage report can be found at',
+            pathlib.Path(log_dir, 'coverage.txt').as_uri())
+    if lcov_exe and genhtml_exe:
+        print('Html coverage report can be found at',
+            pathlib.Path(htmloutdir, 'index.html').as_uri())
```

Comment:
__[Flake8]__

[E128] continuation line under-indented for visual indent


<!-- Comment by [SideCI](https://sideci.com) -->